recipes-tag/py4xs/meta.yaml

```diff
@@ -1,14 +1,14 @@
-{% set version = "2017.12.20" %}
+{% set version = "2019.6.28.0" %}
 {% set name = "py4xs" %}
 
 package:
     name: {{ name }}
     version: {{ version }}
 
+
 source:
     fn: {{ name }}-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/5a/dc/b5f5cf5a607f5de458b870863f3a65dfafaa12421b187619241f8fe4c6cc/{{ name }}-{{ version  }}.tar.gz
-    md5: 783ae4e853e09220e74ba9ebd5d8d1d8
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
```suggestion
    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
    sha256: 75c51a39618f52492a3b3b57465a4f20d1d56005deb53028cb051b8d4e6fbbd7
```

Reply:
I think we should skip the SHA on this recipe.

Reply:
Why?

Reply:
@tacaswell answered in person. We want to rebuild it many times and don't care about hashes for now.